src/trace/coqTerms.mli

```diff
@@ -10,249 +10,322 @@
 (**************************************************************************)
 
 
-val gen_constant : string list list -> string -> CoqInterface.constr lazy_t
```

Comment:
Maybe worth updating the [API](https://github.com/smtcoq/smtcoq-api/blob/4442e18d331829f4d2f1d70853bdaaa160947bd1/src/api.ml#L34-L41) accordingly?

Reply:
SMTCoq-API does not compile with this version of Coq.
A new version is under progress.